public/css/application.css

```diff
@@ -2,25 +2,181 @@
  * Global stylesheet for form elements
  */
 
+#application_logo {
+	position: absolute;
+	top: 170px;
+	left: 50%;
+	padding-left: 30px;
+	padding-right: 30px;
+	transform: translateX(-50%);
+	max-width: 100%;
+}
+
+#intro_graphics {
+	width: 100%;
+	margin-bottom: 20px;
+}
+
+fieldset {
+	padding: 50px 0px;
+	display: flex;
+	align-items: flex-end;
+	justify-content: center;
+	flex-wrap: wrap;
+}
+
+.team_icon {
+	display: flex;
+	flex-wrap: wrap;
+	align-content: center;
+	justify-content: center;
+	padding-bottom: 30px;
+}
+
+.miscellaneous_icon {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+	align-content: center;
+}
+
+.form label {
+	padding-left: 5px;
+}
+
+.form input,
+.form input:focus,
+.form input[readonly] {
+	border-radius: 12px;
+	margin-bottom: 20px;
+	display: inline-block;
+}
+
+.hack-experience {
+	align-content: flex-start;
+	justify-content: space-between;
+	padding: 0px;
+}
+
+.hack-experience {
+	padding: 0px;
+}
+
+/* The container */
+.container {
+	display: inline-block;
+	position: relative;
+	width: 60px;
+	padding-left: 25px;
+	margin: 0;
+	cursor: pointer;
+	font-size: 22px;
+	-webkit-user-select: none;
+	-moz-user-select: none;
+	-ms-user-select: none;
+	user-select: none;
+  }
+  
+  /* Hide the browser's default radio button */
+  .container input {
+	position: absolute;
+	width: 0;
+	opacity: 0;
+	cursor: pointer;
+  }
+  
+  /* Create a custom radio button */
+  .checkmark {
+	position: absolute;
+	top: 0;
+	left: -20px;
+	height: 25px;
+	width: 25px;
+	background-color: #eee;
+	border-radius: 30%;
+  }
+  
+  /* On mouse-over, add a grey background color */
+  .container:hover input ~ .checkmark {
+	background-color: #ccc;
+  }
+  
+  /* When the radio button is checked, add a green background */
+  .container input:checked ~ .checkmark {
+	background-color: var(--green);
+  }
+  
+  /* Create the indicator (the dot/circle - hidden when not checked) */
+  .checkmark:after {
+	content: "";
+	position: absolute;
+	display: none;
+  }
+  
+  /* Show the indicator (dot/circle) when checked */
+  .container input:checked ~ .checkmark:after {
+	display: block;
+  }
+  
+  /* Style the indicator (dot/circle) */
+  .container .checkmark:after {
+	   top: 9px;
+	  left: 9px;
+	  width: 8px;
+	  height: 8px;
```

Comment:
I find, at least on my screen, that having `left: 8.5px` makes the dot better centered